make/migrations/postgresql/0140_2.11.0_schema.up.sql

```diff
@@ -28,4 +28,15 @@ then set column artifact_type as not null
 */
 UPDATE artifact SET artifact_type = media_type WHERE artifact_type IS NULL;
 
-ALTER TABLE artifact ALTER COLUMN artifact_type SET NOT NULL;
\ No newline at end of file
+ALTER TABLE artifact ALTER COLUMN artifact_type SET NOT NULL;
+
+CREATE TABLE IF NOT EXISTS sbom_report
+(
+    id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
let's consider ultimate the uuid as the primary key, rather than the SERIAL PRIMARY KEY.